spec/models/step_by_step_page_spec.rb

```diff
@@ -306,7 +306,7 @@
   end
 
   describe '#can_be_published?' do
-    let(:step_by_step_page) { create(:step_by_step_page_with_steps) }
+    let(:step_by_step_page) { create(:published_step_by_step_page) }
```

Comment:
Is this the correct factory to be using here?  I think a step by step in draft state might be more suited?